### PR TITLE
add ksm core recommendation for helm installation

### PR DESCRIPTION
**Pull request by kennonkwok**
### What does this PR do?

This adds a recommendation for new DD deployments to use KSM Core.

### Motivation

This is considered a best practice and will prevent future migration issues for the customer.

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/containers/kubernetes/installation.md [hunk 1]
```diff
@@ -204,6 +204,21 @@ The Datadog chart has been refactored in v2.0 to regroup the `values.yaml` param
 
 If your current chart version deployed is earlier than `v2.0.0`, follow the [migration guide][11] to map your previous settings with the new fields.
 
+### Kube state metrics core in chart v2.x
+
+In new deployments, Datadog recommends using the newer kube-state-metrics core with the following values:
```
Comment (buraizu) [inline content/en/containers/kubernetes/installation.md:209]:
```suggestion
In new deployments, Datadog recommends using the newer `kube-state-metrics` core with the following values:
```

File: content/en/containers/kubernetes/installation.md [hunk 2]
```diff
@@ -229,6 +244,7 @@ where `<USER_ID>` is the UID to run the agent and `<DOCKER_GROUP_ID>` is the gro
 [9]: https://gallery.ecr.aws/datadog/
 [10]: https://hub.docker.com/u/datadog/
 [11]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/docs/Migration_1.x_to_2.x.md
+[12]: https://docs.datadoghq.com/integrations/kubernetes_state_core
```
Comment (buraizu) [inline content/en/containers/kubernetes/installation.md:247]:
```suggestion
[12]: /integrations/kubernetes_state_core
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks, left a few suggestions as per our [contributing guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md)

**Review by buraizu — CHANGES_REQUESTED** [review]
Thanks for implementing those changes, and apologies -- it looks like I didn't save one of my earlier comments, so I've added it back in.

File: content/en/containers/kubernetes/installation.md [hunk 3]
```diff
@@ -204,6 +204,21 @@ The Datadog chart has been refactored in v2.0 to regroup the `values.yaml` param
...
+```yaml
+...
+datadog:
+...
+  kubeStateMetricsCore:
+    enabled: true
+...
+```
+
+For details about `kube-state-metrics` core, read the [`kube-state-metrics` core documentation][12].
```
Comment (buraizu) [inline content/en/containers/kubernetes/installation.md:220]:
```suggestion
For details about `kube-state-metrics` core, read the [Kubernetes State Metrics Core documentation][12].
```

**Review by buraizu — APPROVED** [review]
Thanks again! I took another look, and I actually think it might be slightly better to have the link show the exact name of the page it opens, so I added that suggestion, but either way I think it should be fine

**Review by johnrk — APPROVED** [review]
LGTM

**Pull request merged**